js/app/cart/CartTop.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class CartTop extends React.Component
+{
+
+  render() {
+    return (
+      <a href={this.props.validateCartURL} className="btn btn-default navbar-btn navbar-right">
```

Comment:
IMHO for now on we better redirect on restaurant page, instead on going for checkout right away when clicking on the top cart. It is because this icon is more used to resume shopping than to go to checkout. Please update `href` accordingly.

Reply:
definitely better, but if we want to do like that we may have to change the <a href=https://github.com/coopcycle/coopcycle-web/blob/677839deb831159a66ff1d368b325a008a068f65/src/AppBundle/Controller/RestaurantController.php#L44> getCart </a> function. Because if the restaurant of the session cart and the one of the current restaurant page are the same then the cart is reseted which is not what we want in this situation.. What do you think ?